### ISSUE TITLE
networkd with 252 with ManageForeignRoutes=no does not configure IPv4 default gateway

### ISSUE THREAD

**Issue by duritong** [systemd/systemd#30403]
### systemd version the issue has been seen with

systemd-252-18.el9.x86_64

### Used distribution

CentOS Stream 9

### Linux kernel version used

5.14.0-390.el9.x86_64

### CPU architectures issue was seen on

x86_64

### Component

networkctl

### Expected behaviour you didn't see

This issue appeared after we upgraded from Centos Stream 8 ( systemd-239-78.el8.x86_64 & systemd-networkd-253.4-1.el8.x86_64) to CentOS Stream 9 (systemd-252-18.el9.x86_64 & systemd-networkd-253.4-1.el9.x86_64)

The same config produces a working config on CentOS Stream 8 but not on CentOS Stream 9.

What we are doing is setting to `ManageForeignRoutes=no` and a network file like the following:

```
[Match]
Name=eth1

[Address]
Address=1.1.8.105/31
Peer=1.1.8.104/31

[Address]
Address=2a02:1234:12:0:432::110:a00f/124

[Route]
Gateway=2a02:1234:12:0:432::110:a001

[Route]
Gateway=1.1.8.104

[Network]
```

This should produce a working network interface with a default gateway for IPv4 and IPv6.

### Unexpected behaviour you saw

However, it does not add a default gateway for IPv4, but only for IPv6:

```
# networkctl status eth1
● 3: eth1
                     Link File: /usr/lib/systemd/network/99-default.link
                  Network File: /etc/systemd/network/20-upstream.network
                         State: routable (configuring)
                  Online state: online
                          Type: ether
                          Path: pci-0000:00:08.0
                        Driver: e1000
                        Vendor: Intel Corporation
                         Model: 82540EM Gigabit Ethernet Controller (PRO/1000 MT Desktop Adapter)
             Alternative Names: enp0s8
              Hardware Address: 08:00:27:c3:37:0a (PCS Systemtechnik GmbH)
                           MTU: 1500 (min: 46, max: 16110)
                         QDisc: fq_codel
  IPv6 Address Generation Mode: eui64
      Number of Queues (Tx/Rx): 1/1
              Auto negotiation: yes
                         Speed: 1Gbps
                        Duplex: full
                          Port: tp
                       Address: 1.1.8.105
                                2a02:1234:12:0:432:0:110:a00f
                                fe80::a00:27ff:fec3:370a
                       Gateway: 2a02:1234:12:0:432:0:110:a001
             Activation Policy: up
           Required For Online: yes
             DHCP6 Client DUID: DUID-EN/Vendor:0000ab11de87d14e5db425f00000
```

Enabling the management of foreign routes, does set `1.1.8.104` as a default IPv4 Gateway.

### Steps to reproduce the problem

The following Vagrantfile allows you to reproduce the issue:

```
$script = <<-'SCRIPT'
dnf install -y epel-release

dnf install -y systemd-networkd

cat >/etc/systemd/networkd.conf<<EOF
[Network]
ManageForeignRoutes=no

[DHCPv4]

[DHCPv6]
EOF

cat >/etc/systemd/network/20-upstream.network<<EOF
[Match]
Name=eth1

[Address]
Address=1.1.8.105/31
Peer=1.1.8.104/31

[Address]
Address=2a02:1234:12:0:432::110:a00f/124

[Route]
Gateway=2a02:1234:12:0:432::110:a001

[Route]
Gateway=1.1.8.104

[Network]
EOF

mkdir  /etc/systemd/system/systemd-networkd.service.d/
cat >/etc/systemd/system/systemd-networkd.service.d/override.conf<<EOF
[Service]
Environment=SYSTEMD_LOG_LEVEL=debug
EOF

systemctl daemon-reload
systemctl restart systemd-networkd

networkctl reload
networkctl reconfigure eth1

sleep 20

journalctl -u systemd-networkd --no-pager

networkctl status eth1

systemctl --version
rpm -qi systemd-networkd

SCRIPT

Vagrant.configure("2") do |config|
  # vagrant box add https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-Vagrant-9-20221219.0.x86_64.vagrant-virtualbox.box --name centos/stream9
  config.vm.box = "centos/stream9"

  config.vm.network "private_network", ip: "192.168.50.4",
    virtualbox__intnet: true, auto_config: false
  config.vm.provision "shell", inline: $script
end
```

### Additional program output to the terminal or log subsystem illustrating the issue

```sh
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: found matching network '/etc/systemd/network/20-upstream.network'.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Configuring with /etc/systemd/network/20-upstream.network.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: LLDP Rx: Stopping LLDP client
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: DHCPv6 lease lost
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Stopping IPv6 Router Solicitation client
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: State changed: initialized -> configuring
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=51 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Setting '/proc/sys/net/ipv6/conf/eth1/disable_ipv6' to '0'
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: No change in value '0', suppressing write
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Setting '/proc/sys/net/ipv6/conf/eth1/use_tempaddr' to '0'
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: No change in value '0', suppressing write
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Setting '/proc/sys/net/ipv6/conf/eth1/accept_ra' to '0'
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: No change in value '0', suppressing write
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Setting '/proc/sys/net/ipv6/conf/eth1/proxy_ndp' to '0'
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: No change in value '0', suppressing write
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Setting '/proc/sys/net/ipv4/conf/eth1/promote_secondaries' to '1'
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: No change in value '1', suppressing write
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Setting '/proc/sys/net/ipv6/conf/eth1/addr_gen_mode' to '0'
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requested to activate link
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requested configuring of the DHCPv6 client.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requested configuring of the IPv6 Router Discovery.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requesting static address (n/a): 1.1.8.105 peer 1.1.8.104/31 (valid forever, preferred forever), flags: tentative, scope: global
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requesting static address (n/a): 2a02:1234:12:0:432:0:110:a00f/124 (valid forever, preferred forever), flags: tentative, scope: global
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Setting addresses
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requesting static route (n/a): dst: n/a, src: n/a, gw: 2a02:1234:12:0:432:0:110:a001, prefsrc: n/a, scope: global, table: main(254), proto: static, type: unicast, nexthop: 0, priority: 1024, flags: n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requesting static route (requesting): dst: n/a, src: n/a, gw: 1.1.8.104, prefsrc: n/a, scope: global, table: main(254), proto: static, type: unicast, nexthop: 0, priority: 0, flags: n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Requesting routes
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): link is not activated.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: LLDP Rx: Started LLDP client
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Bringing link up
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Ignoring received foreign route (configured): dst: fe80::/64, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main(254), proto: kernel, type: unicast, nexthop: 0, priority: 256, flags: n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static addresses are not configured.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: IPv6 link-local address generation mode is changed: none -> eui64
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: DHCPv6 client is configured.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: IPv6 Router Discovery is configured.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Configuring static address (requesting): 1.1.8.105 peer 1.1.8.104/31 (valid forever, preferred forever), flags: tentative, scope: global
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Configuring static address (requesting): 2a02:1234:12:0:432:0:110:a00f/124 (valid forever, preferred forever), flags: tentative, scope: global
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Received updated static address (configured): 1.1.8.105 peer 1.1.8.104/31 (valid forever, preferred forever), flags: permanent, scope: global
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=52 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static addresses are not configured.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/network1 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=53 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Ignoring received foreign route (configured): dst: 1.1.8.105/32, src: n/a, gw: n/a, prefsrc: 1.1.8.105, scope: host, table: local(255), proto: kernel, type: local, nexthop: 0, priority: 0, flags: n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Ignoring received foreign route (configured): dst: 1.1.8.104/31, src: n/a, gw: n/a, prefsrc: 1.1.8.105, scope: link, table: main(254), proto: kernel, type: unicast, nexthop: 0, priority: 0, flags: n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Ignoring received foreign route (configured): dst: 2a02:1234:12:0:432:0:110:a000/124, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: main(254), proto: kernel, type: unicast, nexthop: 0, priority: 256, flags: n/a
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Received updated static address (configured): 2a02:1234:12:0:432:0:110:a00f/124 (valid forever, preferred forever), flags: tentative,permanent, scope: global
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static addresses are not configured.
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: Addresses set
Dec 09 20:52:32 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): address 2a02:1234:12:0:432:0:110:a00f/124 is not ready.
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: eth1: Received updated static address (configured): 2a02:1234:12:0:432:0:110:a00f/124 (valid forever, preferred forever), flags: permanent, scope: global
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=54 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static routes are not configured.
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: Sent message type=signal sender=n/a destination=n/a path=/org/freedesktop/network1 interface=org.freedesktop.DBus.Properties member=PropertiesChanged cookie=55 reply_cookie=0 signature=sa{sv}as error-name=n/a error-message=n/a
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: eth1: Configuring static route (requesting): dst: n/a, src: n/a, gw: 2a02:1234:12:0:432:0:110:a001, prefsrc: n/a, scope: global, table: main(254), proto: static, type: unicast, nexthop: 0, priority: 1024, flags: n/a
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: eth1: Ignoring received foreign route (configured): dst: 2a02:1234:12:0:432:0:110:a00f/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: local(255), proto: kernel, type: local, nexthop: 0, priority: 0, flags: n/a
Dec 09 20:52:33 localhost.localdomain systemd-networkd[3930]: eth1: Received remembered static route (configured): dst: n/a, src: n/a, gw: 2a02:1234:12:0:432:0:110:a001, prefsrc: n/a, scope: global, table: main(254), proto: static, type: unicast, nexthop: 0, priority: 1024, flags: n/a
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.85 destination=org.freedesktop.network1 path=/org/freedesktop/network1 interface=org.freedesktop.DBus.Properties member=Get  cookie=2 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: Sent message type=method_return sender=n/a destination=:1.85 path=n/a interface=n/a member=n/a cookie=56 reply_cookie=2 signature=v error-name=n/a error-message=n/a
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.85 destination=org.freedesktop.network1 path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=Get  cookie=3 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: Sent message type=method_return sender=n/a destination=:1.85 path=n/a interface=n/a member=n/a cookie=57 reply_cookie=3 signature=v error-name=n/a error-message=n/a
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.85 destination=org.freedesktop.network1 path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=Get  cookie=4 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: Sent message type=error sender=n/a destination=:1.85 path=n/a interface=n/a member=n/a cookie=58 reply_cookie=4 signature=s error-name=org.freedesktop.DBus.Error.UnknownProperty error-message=Unknown interface org.freedesktop.network1.DHCPServer or property Leases.
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: Received new foreign address (configured): fe80::a00:27ff:fec3:370a/64 (valid forever, preferred forever), flags: permanent, scope: link
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: Gained IPv6LL
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: Discovering IPv6 routers
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Started IPv6 Router Solicitation client
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static routes are not configured.
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static routes are not configured.
Dec 09 20:52:34 localhost.localdomain systemd-networkd[3930]: eth1: Ignoring received foreign route (configured): dst: fe80::a00:27ff:fec3:370a/128, src: n/a, gw: n/a, prefsrc: n/a, scope: global, table: local(255), proto: kernel, type: local, nexthop: 0, priority: 0, flags: n/a
Dec 09 20:52:35 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Sent Router Solicitation, next solicitation in 3s
Dec 09 20:52:37 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.86 destination=org.freedesktop.network1 path=/org/freedesktop/network1 interface=org.freedesktop.DBus.Properties member=Get  cookie=2 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:52:37 localhost.localdomain systemd-networkd[3930]: Sent message type=method_return sender=n/a destination=:1.86 path=n/a interface=n/a member=n/a cookie=59 reply_cookie=2 signature=v error-name=n/a error-message=n/a
Dec 09 20:52:37 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.86 destination=org.freedesktop.network1 path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=Get  cookie=3 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:52:37 localhost.localdomain systemd-networkd[3930]: Sent message type=method_return sender=n/a destination=:1.86 path=n/a interface=n/a member=n/a cookie=60 reply_cookie=3 signature=v error-name=n/a error-message=n/a
Dec 09 20:52:37 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.86 destination=org.freedesktop.network1 path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=Get  cookie=4 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:52:37 localhost.localdomain systemd-networkd[3930]: Sent message type=error sender=n/a destination=:1.86 path=n/a interface=n/a member=n/a cookie=61 reply_cookie=4 signature=s error-name=org.freedesktop.DBus.Error.UnknownProperty error-message=Unknown interface org.freedesktop.network1.DHCPServer or property Leases.
Dec 09 20:52:39 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Sent Router Solicitation, next solicitation in 7s
Dec 09 20:52:46 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: No RA received before link confirmation timeout
Dec 09 20:52:46 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Invoking callback for 'timeout' event.
Dec 09 20:52:46 localhost.localdomain systemd-networkd[3930]: eth1: NDisc handler get timeout event
Dec 09 20:52:46 localhost.localdomain systemd-networkd[3930]: eth1: link_check_ready(): static routes are not configured.
Dec 09 20:52:47 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Sent Router Solicitation, next solicitation in 15s
Dec 09 20:53:03 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Sent Router Solicitation, next solicitation in 32s
Dec 09 20:53:35 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Sent Router Solicitation, next solicitation in 1min 4s
Dec 09 20:54:16 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.87 destination=org.freedesktop.network1 path=/org/freedesktop/network1 interface=org.freedesktop.DBus.Properties member=Get  cookie=2 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:54:16 localhost.localdomain systemd-networkd[3930]: Sent message type=method_return sender=n/a destination=:1.87 path=n/a interface=n/a member=n/a cookie=62 reply_cookie=2 signature=v error-name=n/a error-message=n/a
Dec 09 20:54:16 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.87 destination=org.freedesktop.network1 path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=Get  cookie=3 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:54:16 localhost.localdomain systemd-networkd[3930]: Sent message type=method_return sender=n/a destination=:1.87 path=n/a interface=n/a member=n/a cookie=63 reply_cookie=3 signature=v error-name=n/a error-message=n/a
Dec 09 20:54:16 localhost.localdomain systemd-networkd[3930]: Got message type=method_call sender=:1.87 destination=org.freedesktop.network1 path=/org/freedesktop/network1/link/_33 interface=org.freedesktop.DBus.Properties member=Get  cookie=4 reply_cookie=0 signature=ss error-name=n/a error-message=n/a
Dec 09 20:54:16 localhost.localdomain systemd-networkd[3930]: Sent message type=error sender=n/a destination=:1.87 path=n/a interface=n/a member=n/a cookie=64 reply_cookie=4 signature=s error-name=org.freedesktop.DBus.Error.UnknownProperty error-message=Unknown interface org.freedesktop.network1.DHCPServer or property Leases.
Dec 09 20:54:37 localhost.localdomain systemd-networkd[3930]: Restart operation initiated.
Dec 09 20:54:37 localhost.localdomain systemd-networkd[3930]: Bus bus-api-network: changing state RUNNING → CLOSED
Dec 09 20:54:37 localhost.localdomain systemd-networkd[3930]: eth1: LLDP Rx: Stopping LLDP client
Dec 09 20:54:37 localhost.localdomain systemd-networkd[3930]: eth1: DHCPv6 lease lost
Dec 09 20:54:37 localhost.localdomain systemd-networkd[3930]: eth1: NDISC: Stopping IPv6 Router Solicitation client
```

**Comment by yuwata** [comment]
Thank you for the report. Fix is waiting in #30404.

**Comment by duritong** [comment]
Thank you for the quick fix!
